ykpack/src/lib.rs

```diff
@@ -210,9 +210,9 @@ mod tests {
         let expect = "symbol: symbol1
   flags: 0
   local_decls:
-    0: (0, 0)
-    1: (0, 0)
-    2: (0, 0)
+    0: 0000000000000000:0
```

Comment:
What's the deal with the long string of zeros in the type id?

Reply:
I used `{:016x}` to represent the crate hash. It is a hash, so in most of the cases it would fill the full 16 hex char even without padding. To keep things aligned when this would not be the case I padded it to 16 hex chars using `0`.

Reply:
OK.

One more question. it used to be `x: (y, z)` but is now `x: y:z`.I can't remember when or why we changed the formatting.

Reply:
I just changed it. It seemed a bit more logical to me, as it is "in crate `y`: type `z`".

Reply:
I can change it back if you prefer the old format.

Reply:
I actually prefer it as it was, as it makes it more obvious it's a tuple.

Sorry to be a bikeshedder.

Reply:
Reverted, squash?